ansible/roles/test/files/helpers/arp_responder.py

```diff
@@ -89,7 +89,11 @@ def action(self, interface):
         if len(data) > self.ARP_PKT_LEN:
             return
 
-        remote_mac, remote_ip, request_ip = self.extract_arp_info(data)
+        remote_mac, remote_ip, request_ip, op_type = self.extract_arp_info(data)
+
+        # Don't send ARP response if the ARP op code is response
+        if op_type == 2:
```

Comment:
I'd better put here
```
if op_type != ARP_OP_REQUEST:
```
op_type could be 3 and 4.

Reply:
Great idea! Done.